src/postman/__tests__/service.test.ts

```diff
@@ -34,6 +34,7 @@ describe('transformPostmanCollectionService()', () => {
               key: 'http-0',
               scheme: 'basic',
               type: 'http',
+              extensions: {},
```

Comment:
there are showing up empty now because extensions is always required?

Reply:
yea - someone made a breaking change in stoplight/types w/o upgrading here